src/coreclr/pal/src/thread/process.cpp

```diff
@@ -3657,6 +3657,7 @@ buildArgv(
         (strcat_s(lpAsciiCmdLine, iLength, " ") != SAFECRT_SUCCESS))
     {
         ERROR("strcpy_s/strcat_s failed!\n");
+        free(lpAsciiCmdLine);
```

Comment:
for anyone else looking - the inconsistency between InternalMalloc vs free (instead of an InternalFree) matches what is defined in malloc.hpp